index.html

```diff
@@ -181,17 +182,24 @@ <h4 class="card__title">Skills</h4>
                 <h2 class="heading-2">I'm always interested in hearing about new projects, so if you'd like to chat, please get in touch.</h2>
             </div>
             <div class="contact__form">
-                <form>
+                <form action="https://formspree.io/f/mrgdrewl"
+                method="POST">
                     <div class="form-control">
-                        <input type="text" placeholder="Full name">
+                        <label>
```

Comment:
- [optional] Great job. the input fields don't need to be rapped in different label tags.the label tag is needed if you want to name your iput field. check [here ](https://www.w3schools.com/html/html_forms_attributes.asp)for more